### ISSUE TITLE
[DevTools Bug] Unsupported Bridge operation "0"

### ISSUE THREAD

**Issue by tuanbmfsoft** [facebook/react#24142]
### Website or app

React Native Init App

### Repro steps

Just run the React-DevTools and then forward to port 8087 to debug in real device

### How often does this bug happen?

Every time

### DevTools package (automated)

react-devtools-core

### DevTools version (automated)

4.24.1-ac574d688

### Error message (automated)

Unsupported Bridge operation "0"

### Error call stack (automated)

```text
at /Users/tb921t/.npm-global/lib/node_modules/react-devtools/node_modules/react-devtools-core/dist/standalone.js:48:335334
    at f.emit (/Users/tb921t/.npm-global/lib/node_modules/react-devtools/node_modules/react-devtools-core/dist/standalone.js:48:278775)
    at /Users/tb921t/.npm-global/lib/node_modules/react-devtools/node_modules/react-devtools-core/dist/standalone.js:48:280316
    at /Users/tb921t/.npm-global/lib/node_modules/react-devtools/node_modules/react-devtools-core/dist/standalone.js:48:664831
    at Array.forEach (<anonymous>)
    at A.Zh.e.onmessage (/Users/tb921t/.npm-global/lib/node_modules/react-devtools/node_modules/react-devtools-core/dist/standalone.js:48:664815)
    at A.t (/Users/tb921t/.npm-global/lib/node_modules/react-devtools/node_modules/react-devtools-core/dist/standalone.js:39:2836)
    at A.emit (events.js:315:20)
    at e.exports.L (/Users/tb921t/.npm-global/lib/node_modules/react-devtools/node_modules/react-devtools-core/dist/standalone.js:3:58322)
    at e.exports.emit (events.js:315:20)
```

### GitHub query string (automated)

```text
https://api.github.com/search/issues?q=Unsupported Bridge operation  in:title is:issue is:open is:public label:"Component: Developer Tools" repo:facebook/react
```

**Comment by tuanbmfsoft** [comment]
I'm using the DevTools v4.24.1.
React-Native: 0.6.7

**Comment by suwu150** [comment]
I'm using the DevTools v4.22.0.
React-Native: 0.67.3

also fail.

**Comment by bvaughn** [comment]
@tuanbmfsoft To the best of my knowledge, 0.6.7 is not a valid version for React Native. I assume you meant 0.67?

**Comment by bvaughn** [comment]
@tuanbmfsoft To the best of my knowledge, 0.6.7 is not a valid version for React Native. I assume you meant 0.67?

@tuanbmfsoft @suwu150 Can you clarify what is meant by "React Native Init App" ?

Are you referring to the `expo init` command? If so, I am unable to reproduce using `expo init` and React DevTools 4.24.1. That combination works for me.

Can you provide clearer repro steps? Without them, this issue isn't something we can act on.

**Comment by bvaughn** [comment]
> I'm using the DevTools v4.22.0.
React-Native: 0.67.3

@suwu150 Please update to the latest version of React DevTools (4.24.1 at this time).

If that does not work, please share full repro instructions with us so that we can look into what's going wrong.

**Comment by tuanbmfsoft** [comment]
Hi @bvaughn 
I just create a project using react-native init and got this error.
 i’m not using expo. And it’s 0.67 😄 
Thank you for ur support.
Tuan

**Comment by bvaughn** [comment]
Thanks!

Okay to confirm: the repro requires using a new frontend (e.g. 4.24) with an older backend (e.g. 4.19).

**Comment by tuanbmfsoft** [comment]
Hi @bvaughn 
Just downgrade the devtools back to 4.19 and it working now.
Thank you.
Tuan

**Comment by bvaughn** [comment]
That works! Alternately you could upgrade `react-devtools-core` (the `react-native` dependency) to a newer version.

**Comment by EmmanuelCanto95** [comment]
> Hi @bvaughn Just downgrade the devtools back to 4.19 and it working now. Thank you. Tuan

Same here! Thanks 😁

**Comment by bvaughn** [comment]
Glad to hear that you've both got this sorted out.

We are considering ways to improve the user experience on our end, but there might not be a great solution to this for older backend versions. Fortunately, newer React Native versions will depend on newer DevTools backends and this issue will gradually go away.

**Comment by bvaughn** [comment]
I think the best pragmatic thing we can do in this case would be to update the error UI in DevTools for this specific case so that, rather than showing something like this:
<img width="778" alt="Screen Shot 2022-03-23 at 3 50 01 PM" src="https://user-images.githubusercontent.com/29597/159788846-4f6db622-c97e-4448-98de-0b32c39f7629.png">

We show something like this:
<img width="830" alt="Screen Shot 2022-03-23 at 4 21 49 PM" src="https://user-images.githubusercontent.com/29597/159788847-bd6bc9a1-2c44-4a44-8787-f8b00c91575e.png">

**Comment by bvaughn** [comment]
I think we've done the best we can with this issue unfortunately. Going to close it.

**Comment by bvaughn** [comment]
The above UI improvement has been published as version [4.24.2](https://github.com/facebook/react/blob/main/packages/react-devtools/CHANGELOG.md#4242).
